HomeWork9/HomeWork9/Services/FileService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Services.Abstracts;
+
+namespace Services
+{
+    internal class FileService : IFileService
+    {
+        private readonly static string str;
+        private const string _path = $"C:\\Users\\Сім'я\\source\\repos\\HomeWork9\\HomeWork9\\Log\\";
+
+        static FileService()
+        {
+            DeleteOverFile();
+
+            str = DateTime.Now.ToString("MM/dd/yyyy hh.mm.ss.ffftt");
```

Comment:
UtcNow

Reply:
use now because different time UNcNow and Now in my logs)) Now show right time)) but i allready now diferents)